app/controllers/introduce_yourself_controller.rb

```diff
@@ -17,6 +17,10 @@ def update
 
   private
 
+  def ensure_application_present
+    _application_not_created = true
+  end
```

Comment:
ahh ok -- so I think the var here was actually creating more confusion for me. Just return `true`?